snapcraft/internal/parts.py

```diff
@@ -87,5 +87,44 @@ def _save_headers(self):
             headers_file.write(yaml.dump(headers))
 
 
+class _RemoteParts(_Base):
+
+    def __init__(self):
+        super().__init__()
+
+        if os.path.exists(self.parts_yaml):
+            with open(self.parts_yaml) as parts_file:
+                self._parts = yaml.load(parts_file)
+        else:
+            self._parts = {}
+
+    def get_part(self, part_name):
+        remote_part = self._parts[part_name].copy()
+        for key in ['description', 'maintainer']:
+            remote_part.pop(key)
+        return remote_part
+
+    def compose(self, part_name, properties):
+        """Return properties composed with the ones from part name in the wiki.
+        :param str part_name: The name of the part to query from the wiki
+        :param dict properties: The current set of properties
+        :return: Part properties from the wiki composed with the properties
+                 passed as a parameter. If there is no wiki part named name,
```

Comment:
What about:
If there is no wiki part with the name passed as parameter, just the properties parameter will be returned.